test/test_helper.rb

```diff
@@ -81,7 +81,9 @@ def create_items_associated_with_orders
   end
 
   def create_user
-    User.create(username: 'Matt', password: 'gnargnar')
+    User.create(username: 'Matt',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.